docs/OV_Runtime_UG/migration_ov_2_0/intro.md

```diff
@@ -0,0 +1,81 @@
+# OpenVINO™ 2.0 Transition Guide {#openvino_2_0_transition_guide}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   
+   openvino_2_0_inference_pipeline
+   openvino_2_0_configure_devices
+   openvino_2_0_preprocessing
+   openvino_2_0_model_creation
+      
+@endsphinxdirective
+
+### Introduction
+
+The OpenVINO™ 2.0 is introduced to simplify migration of user applications from the frameworks like TensorFlow, PyTorch, ONNX, etc to OpenVINO runtime and make the OpenVINO™ API more user-friendly. This includes changes in several OpenVINO components:
```

Comment:
We don't have native PyTorch support, maybe will be better to change to PaddlePaddle

Reply:
here I am talking not about directly supported model formats, about the Frameworks at all.